### PR TITLE
Fix e2e add external service

### PR DESCRIPTION
**Pull request by lguychard**
Fix for https://github.com/sourcegraph/sourcegraph/issues/7212#issuecomment-566781691

### CODE REVIEW

File: shared/src/e2e/driver.ts [hunk 1]
```diff
@@ -235,24 +235,39 @@ export class Driver {
...
+                    query ExternalServices {
+                        externalServices(first: 1) {
+                            totalCount
+                        }
+                    }
+                `,
+                variables: {},
+            })
+        )
+        // Delete exsting external services if there are any.
```
Comment (lguychard) [inline shared/src/e2e/driver.ts]:
```suggestion
        // Delete existing external services if there are any.
```

**Comment by eseliger** [comment]
hmm are the e2e tests revealing a 2nd source of failure now? 🤔

**Comment by attfarhan** [comment]
@eseliger I just hit rebuild, let's see if it happens again. It seems like it wasn't able to access localhost:7080 at all previously.

**Comment by eseliger** [comment]
not sure how the timeouts can happen but this seems to fix at least the biggest issue in e2e tests, great find :)

**Comment by attfarhan** [comment]
The e2e tests passed on rebuild. I think the initial failure was `socat` failing, not sure why that happened though.

**Comment by attfarhan** [comment]
Going to merge this since it has two approvals

**Pull request merged**